geode-core/src/main/java/org/apache/geode/internal/ObjectInputStreamFilterWrapper.java

```diff
@@ -81,25 +81,6 @@ public ObjectInputStreamFilterWrapper(String serializationFilterSpec,
       }
     }
 
-    try {
-      URL sanctionedSerializables = ClassPathLoader.getLatest()
-          .getResource(InternalDataSerializer.class, "sanctioned-geode-core-serializables.txt");
-      Collection<String> coreClassNames =
-          InternalDataSerializer.loadClassNames(sanctionedSerializables);
-
-      URL sanctionedManagementSerializables = ClassPathLoader.getLatest()
-          .getResource(InternalDataSerializer.class,
-              "sanctioned-geode-management-serializables.txt");
-      Collection<String> managmentClassNames =
-          InternalDataSerializer.loadClassNames(sanctionedManagementSerializables);
-
-      sanctionedClasses.addAll(coreClassNames);
-      sanctionedClasses.addAll(managmentClassNames);
-    } catch (IOException e) {
-      throw new InternalGemFireException(
-          "unable to read sanctionedSerializables.txt to form a serialization acceptlist", e);
-    }
-
```

Comment:
Note: This special-case section is no longer needed.

CoreSanctionedSerializablesService now exists in geode-core so that the above for-loop loads "sanctioned-geode-core-serializables.txt".

ManagementSanctionedSerializablesService now exists in geode-management so that the above for-loop loads "sanctioned-geode-management-serializables.txt".